ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.hpp

```diff
@@ -164,7 +163,7 @@ std::vector<Tensor> _hardtanh_bw( const Tensor& grad, const Tensor& input, float
 
 std::vector<Tensor> _add_bw( const Tensor& grad, const Tensor& input, float alpha, const std::optional<MemoryConfig>& output_mem_config = std::nullopt);
 std::vector<Tensor> _eq_bw( const Tensor& grad, const Tensor& input, float other, const std::optional<MemoryConfig>& output_mem_config = std::nullopt);
-
+std::vector<Tensor> _remainder_bw( const Tensor& grad, const Tensor& input, float value, const std::optional<MemoryConfig>& output_mem_config);
```

Comment:
Added in unary?

Reply:
@Aswinmcw 
I used add_bw as reference (it is given in unary as well, might need a code cleanup).